cluster/manifests/node-monitor/daemonset.yaml

```diff
@@ -45,7 +45,7 @@ spec:
             - --docker_only
             - --raw_cgroup_prefix_whitelist=/system.slice/kubelet.service
             - --store_container_labels=false
-            - --whitelisted_container_labels=io.kubernetes.container.name,io.kubernetes.pod.name,io.kubernetes.pod.namespace,io.kubernetes.pod.uid,application
+            - --whitelisted_container_labels=io.kubernetes.container.name,io.kubernetes.pod.name,io.kubernetes.pod.namespace,io.kubernetes.pod.uid,application,component
```

Comment:
Why do we need this change? I'm just curious

Reply:
This is such that we can match exactly on `application=zmon-appliance,component=worker` when we exclude it in the scraping. Without this we would exclude all zmon-appliance pods, also those that are not workers which we don't need/want to do.

It's also anyway helpful to have the metrics tagged with component because it's a standardized label (in Zalando) that we just forgot to include.